### PR TITLE
Orca: add get remote file to local decorator in orca file utils.

### PR DESCRIPTION
**Pull request by lalalapotter**
## Description
This PR is to add python implemented decorator for multiple FS loading without PyArrow.

### CODE REVIEW

**Comment by jason-dai** [comment]
Do you have a unit test?

**Comment by lalalapotter** [comment]
Sure, I will add an unit test.

File: python/orca/src/bigdl/orca/data/file.py [hunk 1]
```diff
@@ -537,3 +537,31 @@ def get_remote_files_with_prefix_to_local(remote_path_prefix, local_dir):
         except Exception as e:
             invalidOperationError(False, str(e), cause=e)
     return os.path.join(local_dir, prefix)
+
+
+def enable_multi_fs_load(load_func):
```
Comment (hkvision) [inline python/orca/src/bigdl/orca/data/file.py:542]: rename to multi_fs_load

File: python/orca/src/bigdl/orca/data/file.py [hunk 2]
```diff
@@ -537,3 +537,31 @@ def get_remote_files_with_prefix_to_local(remote_path_prefix, local_dir):
         except Exception as e:
             invalidOperationError(False, str(e), cause=e)
     return os.path.join(local_dir, prefix)
+
+
```
Comment (hkvision) [inline python/orca/src/bigdl/orca/data/file.py:541]: Add some comment to tell the difference between the one in dllib and the one here.
For example, this decorator can load on each worker while that can only load on driver

File: python/orca/src/bigdl/orca/data/file.py (hunk 1 above)
Comment (lalalapotter) [reply python/orca/src/bigdl/orca/data/file.py]: fixed.

File: python/orca/src/bigdl/orca/data/file.py (hunk 2 above)
Comment (lalalapotter) [reply python/orca/src/bigdl/orca/data/file.py:541]: Added.

File: python/orca/src/bigdl/orca/data/file.py [hunk 3]
```diff
@@ -537,3 +537,43 @@ def get_remote_files_with_prefix_to_local(remote_path_prefix, local_dir):
...
+def multi_fs_load(load_func):
+    """
+
+    Enable loading file or directory in multiple file systems.
+    It supports local, hdfs, s3 file systems.
+    Note: this decorator is different from dllib decorator @enable_multi_fs_load.
+    This decorator can load on each worker while @enable_multi_fs_load can only load on driver.
+
+    :param load_func: load file or directory function
+    :return: load file or directory funtion for specific file system
```
Comment (hkvision) [inline python/orca/src/bigdl/orca/data/file.py:551]: typo `function`
for `the`

Comment (lalalapotter) [reply python/orca/src/bigdl/orca/data/file.py]: fixed.

**Review by hkvision — APPROVED** [review]

**Pull request merged**